public/_posts/_updates/web-team/yearn-web-engineering-update-2022-01-31/es.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "Actualización de ingeniería web: 31 de enero de 2022"
+categories: [Updates]
+image:
+  src: ./logo.png
+  width: 1256
+  height: 1024
+author: f0xTheSin & dudesahn
+date: "2022-02-04"
+translator: RestlessMik3
+---
+
+fuente original: [Web Engineering Update 31 Jan](https://yearnweb.substack.com/p/yearn-web-engineering-update-160?s=r)
+
+#### Semana del 31 de enero de 2022
+
+## **Resumen**
+
+Algunas semanas de limpieza mientras terminamos enero de 2022:
+
+- Se ha incluido soporte para Arbitrum en un lanzamiento reciente para el sitio principal, así como para YearnWatch. Con algunas pruebas adicionales, Yearn estará listo para lanzar vaults y estrategias en nuestra tercera cadena.
+- El equipo web también enfrentó algunos problemas de crecimiento cuando abordamos el mayor aumento de visitas de usuarios únicos que jamás hayamos tenido. Esto generó el crecimiento de un equipo de infraestructura con más detalles por venir.
+- Web y estrategas se unieron para comenzar a revisar nuestro APY: agregar más datos, hacerlo más útil para los usuarios y hacerlo más accesible visualmente.
+- Finalmente, hemos tenido un buen impulso de nuevos colaboradores que ayudaron con problemas en varios repositorios web.
+
+## **Logros ✅**
+
+**SDK Versión 1.0.28**
+
+- logro: admite múltiples estrategias en los detalles de la vault
+- corrección: actualice la dirección del ejemplo de simulación
+- corrección: migrar a un nuevo punto final de caché
```

Comment:
AS-IS: corrección: migrar a un nuevo punto final de caché
TO-BE; corrección: migrar a un nuevo endpoint de caché


Endpoint is a technical term for APIs. LEt's keep it in english, the technical ppl that will read these contents will understand the term "endpoint".

Reply:
Perfect, thansks, done